File: ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecurePrepareLookupCallback.java
package com.braintreepayments.api;

import android.content.Context;

/**
 * Callback for receiving result of
 * {@link ThreeDSecureClient#prepareLookup(Context, ThreeDSecureRequest,
 * ThreeDSecurePrepareLookupCallback)}.
 */
public interface ThreeDSecurePrepareLookupCallback {

    /**
     * @param prepareLookupResult {@link ThreeDSecurePrepareLookupResult}
     */
    void onPrepareLookupResult(ThreeDSecurePrepareLookupResult prepareLookupResult);
}

File: ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecurePrepareLookupCallback.kt
package com.braintreepayments.api

/**
 * Callback for receiving result of [ThreeDSecureClient.prepareLookup].
 */
interface ThreeDSecurePrepareLookupCallback {

    /**
     * @param prepareLookupResult [ThreeDSecurePrepareLookupResult]
     */
    fun onPrepareLookupResult(prepareLookupResult: ThreeDSecurePrepareLookupResult?)
}
